schema_based_on_diagram.sql

```diff
@@ -0,0 +1,60 @@
+-- Create patients table
+
+CREATE TABLE patients (
+  id INT,
+  name VARCHAR(255),
+  date_of_birth DATE,
+  PRIMARY KEY(id)
+);
+
+-- Create invoices table
+
+CREATE TABLE invoices (
+  id INT,
+  total_amount DECIMAL(10, 2),
+  generated_at TIMESTAMP,
+  payed_at TIMESTAMP,
+  medical_history_id INT REFERENCES medical_histories(id),
+  PRIMARY KEY(id)
+);
+
+-- Medical histories table 
+
+CREATE TABLE medical_histories (
+  id INT,
+  admitted_at TIMESTAMP,
+  patient_id INT,
```

Comment:
- [ ] Please your FOREIGN KEYS are set by using the `refrences table_name(id)` in column definition.
```
patient_id INT REFERENCES patient(id)
```